src/allomancy/java/leaf/cosmere/allomancy/common/manifestation/AllomancyBrass.java

```diff
@@ -27,10 +34,35 @@ public AllomancyBrass(Metals.MetalType metalType)
 
 	//Dampens Emotions
 	@Override
-	protected void applyEffectTick(ISpiritweb data)
+	public void applyEffectTick(ISpiritweb data)
+	{
+		if (data.getLiving().level.isClientSide())
+		{
+			boolean isSingleTarget = Keybindings.MANIFESTATION_USE_ACTIVE.isDown() || AllomancyKeybindings.ALLOMANCY_SOOTHE.isDown();
```

Comment:
For the first keybind check, need to check spiritweb selected manifestation is the same as this manifestation

Reply:
Replaced with `(data.getSelectedManifestation().equals(getManifestation()) && Keybindings.MANIFESTATION_USE_ACTIVE.isDown())`, which I assume is what was meant by this. Same in Zinc :)

Reply:
that's correct :)

Reply:
Although I would put the keybind in first, since there's no need to check current manifestation if the user isn't holding the keybind down. That's more of a nitpick though, since the performance will largely be the same.

Reply:
I'll switch it around, no worries. It's probably best if the specific keybind check is first as well, since then the generic keybind isn't necessary

Reply:
Done :)